allennlp/commands/find_learning_rate.py

```diff
@@ -295,7 +244,7 @@ def search_learning_rate(trainer: Trainer,
             logger.info(f'Loss ({loss}) exceeds stopping_factor * lowest recorded loss.')
             break
 
-        trainer.rescale_gradients()
+        rescale_gradients(trainer.model, trainer._grad_norm) # pylint: disable=protected-access
```

Comment:
if you're asking why `rescale_gradients` is a bare function rather than a Trainer instance method, it's so that anyone can use it as part of their trainer implementation. But I agree this is ugly, so I added a trainer instance method that wraps it.